### DISCUSSION TITLE
Current BRICKING Status.... 350 & M1

### DISCUSSION THREAD

**Discussion by highveld66** [GerryDazoo/Slinger#437] [Q&A] [answered]
Hello can you advise me on the current status for using 350 or M1 slingboxes

Are they being bricked?
Do you need to block connections for these models?
If so what do you connect and in or out or both?

Bottom line is , is it safe to use slinger and my slingboxes?

Thanks for your help

**Comment by floormanager1** [comment, marked as answer]
I have a 350 and M1 working with Slinger.  I block all internet traffic to / from them as my router will not allow me to easily block a single port.

**Reply by mdehrlich** [reply]
Assuming slinger and your slingboxes are running on your local network, the best thing is completely block the slingboxes in your router from any in or outbound internet traffic.  Use their ip addresses and block them in your router firewall.

**Reply by floormanager1** [reply]
Yes they are.  All works perfectly and I can watch away from my home, which is what I want.

**Reply by highveld66** [reply]
Just wondering if there are no firewall rules in place will a 350 or a m1 get bricked?  Just trying to understand the current position.
thanks

**Reply by floormanager1** [reply]
Well, I had 5 boxes live on the 9th of Nov.  The Pro, Solo and Pro HD got bricked, the M1 and 350 survived.  I believe that M1s and 350s have not yet been bricked but why take the risk?  With the Slinger on the same network as the the M1 and 350, then you can block internet assess to the slingboxes.  The slinger is all you need unblocked to watch away from home.

**Reply by GerryDazoo** [reply]
### ProHD should be recoverable.

**Reply by floormanager1** [reply]
Yes, it was :-)

**Reply by highveld66** [reply]
Thanks for the info
Will add a set of rules to block internet for MAC and belt/braces ip of my Slingboxes

**Reply by mdehrlich** [reply]
I do not think we have any report of a 350/M1/M2 or 500 being bricked.  Biggest problem is that the boxes send messages to sling servers  and they might want to repurpose these ip addresses someday.  It means they might try to brick any boxes still sending them messages.  It's safer to do the firewall block once you have slinger working.

**Reply by highveld66** [reply]
yes makes sense that this could be expoloited if they want to.
Shame we have to think like this.
Would be nice if they put a statement out clarifying their position on this whole saga
In the mean time the insurance policy of firewall rules will be used.
Thanks again to all for their comments/help

**Reply by al55442** [reply]
I think my 350 might have been bricked yesterday....I have had the slinger running for about 3 weeks....
Then yesterday my 350 ip is only visible on my Linksys router for about 1 minute after I power cycle the 350 and listed as Network Device.
I did not have any firewall running to block the internet packets....

**Reply by GerryDazoo** [reply]
Bricked devices are still visible on the network. You have something else wrong.

**Reply by al55442** [reply]
Plugged the 350 in my main router....worked fine.....found a axillary router that was generating bad IP addresses...... on some devices most likely someone trying to hack in ....... cleared the router and changed all passwords on router....so far working ...in the process of hooking up devices to that router to see of the problem returns..

**Comment by robosmith** [comment]
I bought a 500 on ebay, and got it to work with Slinger running on my notebook. However, I don't see a way to finish the setup so it doesn't need to be repeated every time it's turned off. As part of setup, it apparently contacts the sling servers to get a list of set top boxes from which to select one. However, it never finds a list and it won't proceed to the end from there. Also, it needs Internet access to verify the networking is connected at the beginning.

Resetting up the box every time it's turned off is a PITA. Any body know a way to complete setup?

**Reply by Mover480** [reply]
Nope, after it gets the internet connection, I exit the setup and just leave the box on.  Other than the SB logo pulsing on the front of the box, it seems to work just fine.

**Reply by GerryDazoo** [reply]
Robo, Other people have reported this. How are you doing the setup? SlingPlayerDesktop 5.0.0.83?
All I can say it works for me. In this case the host running SPD must have access to the Sling servers but it's OK to keep the slingbox firewalled.

**Reply by robosmith** [reply]
@GerryDazoo I am using a Harmony remote to enter the (limited) setup info, per the video. I installed SlingPlayerDesktop 5.0.0.83, but IIRC, it just gave me a black screen with unfettered Internet access. Does it allow setup of the 500 to complete and save for you?

**Reply by robosmith** [reply]
Ok, I ran SlingPlayerDesktop again, and it apparently found my SB500 on the network, but just spins a wait circle indefinitely. ;(

**Reply by GerryDazoo** [reply]
You can kill it and try after 10 seconds or some. If it persists, I'd check that you have not blocked access to the sling servers for the host where you are running the SPD. You should see a login screen.

**Reply by robosmith** [reply]
@GerryDazoo I created an account when I first ran SDP. I have already checked and removed the blocking for all the sling server sites. The only way I see to kill the spinning wait circle is to terminate the SDP and start over. Strange that sling lets me create an account but does not show a login screen now.

**Reply by Mover480** [reply]
I don't have any sites blocked either and there is no way that SPD will do anything.  Whatever servers it is trying to access are shut down.

**Reply by robosmith** [reply]
@GerryDazoo what is your starting point when running SPD? Factory reset? All I see is the "Initializing...." spinning circle.

**Reply by GerryDazoo** [reply]
I run it and it logs in and I can edit boxes
![image](https://user-images.githubusercontent.com/102599361/209243613-49b5e2f2-8592-4d9b-af95-fef0fb69c61b.png)

![image](https://user-images.githubusercontent.com/102599361/209243738-ad841694-b185-46f0-bd2f-10730415b820.png)

**Reply by robosmith** [reply]
@GerryDazoo did you have the account and admin pw for those pre-9-12? Do a factory reset on the M1 or 500?
I factory reset the 500 and when it connected, it offered to update the firmware. Now when I boot it up, the starting point for the HDMI output is a screen that says "Slingbox Updated"  and "press the onscreen NEXT button to complete setup." With SPD, I just discovered how to see the "Slingbox Directory" box. While it is searching for a slingbox, I press the close button at the top and get a screen that says "Please connect to a slingbox." I click connect button and it connects to "My Slingbox" but when I click the "setup" button it either goes to the "Inititializing...." spin disk or I get large sign with an Error code (-21 or -118) and a msg to check the Internet connection. I can ping sling.com and slingbox.com, but no response from slingmedia.com. Well, at least I see some indication that SPD is able display the output of the 500 box stream even if I still cannot set it up to completion.

**Reply by GerryDazoo** [reply]
Yes, I had an account preN9. Doesn't everybody? I was going to ask why to tried to create a new account.
I did not do a factory reset, I knew the existing password. Hopefully you'll be able to recover the new password for your box. Many sites on the web are non-pingable as long as the name resolves to and IP address it should work. I can assure you it is up and running. But that doesn't mean you don't have it blocked somehow.
